### ISSUE TITLE
Font-awesome icons no longer loading in v4_0_45

### ISSUE THREAD

**Issue by meekoH** [wet-boew/cdts-sgdc#616]
On our web application, we have been using the font-awesome library packaged with the GCWeb CDTS v4.0.45 for our icon sets. These worked perfectly fine up until about a month ago, but are now no longer loaded by the CDTS. Upon downgrading to v.4.0.44, the icons come back and work as intended.

CDTS scripts initially used (font-awesome library not loading):
```
<script src="https://www.canada.ca/etc/designs/canada/cdts/gcweb/rn/cdts/compiled/soyutils.js"></script>
<script src="https://www.canada.ca/etc/designs/canada/cdts/gcweb/rn/cdts/compiled/wet-en.js"></script>
``` 

CDTS scripts I tested to specify v4.0.45 (font-awesome library not loading):
```
<script src="https://www.canada.ca/etc/designs/canada/cdts/gcweb/v4_0_44/cdts/compiled/soyutils.js"></script>
<script src="https://www.canada.ca/etc/designs/canada/cdts/gcweb/v4_0_44/cdts/compiled/wet-en.js"></script>
<script src="https://www.canada.ca/etc/designs/canada/cdts/gcweb/v4_0_44/cdts/compiled/plugins-en.js"></script>
```

CDTS scripts I tested for v4.0.44 (font-awesome library loads correctly):
```
<script src="https://www.canada.ca/etc/designs/canada/cdts/gcweb/v4_0_44/cdts/compiled/soyutils.js"></script>
<script src="https://www.canada.ca/etc/designs/canada/cdts/gcweb/v4_0_44/cdts/compiled/wet-en.js"></script>
<script src="https://www.canada.ca/etc/designs/canada/cdts/gcweb/v4_0_44/cdts/compiled/plugins-en.js"></script>
```

Example of v4.0.44 with font-awesome working as intended:
![image](https://user-images.githubusercontent.com/5738362/192591057-6bfc3ef7-4544-485b-8f6c-a4ae965d1aab.png)

Example of v4.0.45 without font-awesome working:
![image](https://user-images.githubusercontent.com/5738362/192591292-dcb2e392-e737-4bce-b246-5778f65d05a2.png)

It would be nice to not have to downgrade our CDTS version, however, any suggested solutions would be appreciated!

Cheers, and thank you kindly.

**Comment by plup2** [comment]
Yes, fontawesome was intentionally removed as part of the 4.0.45 release (mentioned in [CHANGELOG](https://github.com/wet-boew/cdts-sgdc/blob/master/CHANGELOG.md) - the .45 release was done in May but only recently was made the `/rn/` version).  Main reason for the removal was that CDTS itself does not use fontawesome for anything and yet was pulling it in many, many applications that don't use it either (we weren't even aware of any application that actually used these fonts until now).  Secondary reason is that as an external dependency it was causing problems for an application behind a network with hardened security.

We currently have no plan to bring it back unless there is important demand for it, but you can easily add it back to your app by including the following in your page's `<head>`:
```html
<link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.8.1/css/all.css" integrity="sha384-50oBUHEmvpQ+1lW4y57PTFmhCaXp0ML5d60M1M7uH2+nqUivzIebhndOJK28anvf" crossorigin="anonymous">
```
(this is the only fontawesome-related line we removed from CDTS' source code)

Sorry for the inconvenience.  Let us know if this solution works for you.
